<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WifiScreen</class>
 <widget class="QDialog" name="WifiScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(30, 30, 30, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 255);
font: 24px &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Wi-Fi Settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_logo">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/rsrc/logo.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>60</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(200, 200, 150, 255);
background: qradialgradient(cx:.4, cy:.2, radius:1, fx:.5, fy:.3, stop:.2 rgba(70, 70, 70, 255), stop:.9 rgba(20, 20, 20, 255));
border-style: solid;
border-width: 3px;
border-radius: 30px;
border-color: rgba(115, 115, 95, 255);
font: 18px &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>411</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_status">
      <property name="styleSheet">
       <string notr="true">color: rgba(255, 255, 255, 255);
font: 18px &quot;DejaVu Sans&quot;;</string>
      </property>
      <property name="text">
       <string>Connection Status:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_ap">
      <property name="styleSheet">
       <string notr="true">color: rgba(255, 255, 255, 255);
font: 18px &quot;DejaVu Sans&quot;;</string>
      </property>
      <property name="text">
       <string>Current AP:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_ipaddr">
      <property name="styleSheet">
       <string notr="true">color: rgba(255, 255, 255, 255);
font: 18px &quot;DejaVu Sans&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;IP Address:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="changeButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>60</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(200, 200, 150, 255);
background: qradialgradient(cx:.4, cy:.2, radius:1, fx:.5, fy:.3, stop:.2 rgba(70, 70, 70, 255), stop:.9 rgba(20, 20, 20, 255));
border-style: solid;
border-width: 3px;
border-radius: 30px;
border-color: rgba(115, 115, 95, 255);
font: 18px &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Change AP</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>WifiScreen</receiver>
   <slot>slot1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>changeButton</sender>
   <signal>clicked()</signal>
   <receiver>WifiScreen</receiver>
   <slot>slot2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot1()</slot>
  <slot>slot2()</slot>
 </slots>
</ui>
